src/Frontend/Utils/StockConfigs.ts

```diff
@@ -0,0 +1,435 @@
+import { LobbyInitializers } from "../Panes/Lobbies/Reducer";
```

Comment:
Also, do we want to turn on zk checks? Or not for beta purposes?

Reply:
Also, we should set start paused to false.  It's annoying for new players.

Reply:
did all of this

Reply:
Storing a list of config notes here, but we can come back and edit after this PR:
- Photoid activation time should be 1 minute
-

Reply:
Also, I think it would be ideal to get the deterministic lobby address by parsing transaction event logs instead of listening, in case too many people make lobbies at the same time and the listener gets confused.

https://github.com/ethers-io/ethers.js/issues/179